Winit version conflict in gfx_window_metal.

With the following dependencies:

```
winit = "0.6.0"
gfx = "0.16"
gfx_core = "0.7.1"
glutin = "0.8.0"
gfx_device_gl = "0.14.1"
gfx_window_glutin = "0.16.0"
gfx_window_metal = "0.2.0"
```

I get the following errors at compiltion:

```
error[E0308]: mismatched types
  --> src/core/application.rs:47:109
   |
47 |                 let (window, mut device, mut factory, color) = gfx_window_metal::init::<gfx::format::Rgba8>(winit_builder);
   |                                                                                                             ^^^^^^^^^^^^^ expected struct `winit::WindowBuilder`, found a different struct `winit::WindowBuilder`
   |
   = note: expected type `winit::WindowBuilder` (struct `winit::WindowBuilder`)
              found type `winit::WindowBuilder` (struct `winit::WindowBuilder`)
note: Perhaps two different versions of crate `winit` are being used?
  --> src/core/application.rs:47:109
   |
47 |                 let (window, mut device, mut factory, color) = gfx_window_metal::init::<gfx::format::Rgba8>(winit_builder);
   |                                                                                                             ^^^^^^^^^^^^^
```

It looks like `gfx_window_metal` was built against `winit@0.6`. Did a patch version break compatibility or am I screwing up the dependencies somehow?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,8 +23,8 @@ name = "gfx_app"
 [dependencies]
 log = "0.3"
 env_logger = "0.4"
-glutin = "0.8.0"
-winit = "0.6.0"
+glutin = "0.8"
+winit = "0.6"
 gfx_core = { path = "src/core", version = "0.7.1" }
 gfx_corell = { path = "src/corell", version = "0.1" }
 gfx = { path = "src/render", version = "0.16" }
@@ -54,7 +54,7 @@ optional = true
 
 [dependencies.gfx_window_metal]
 path = "src/window/metal"
-version = "0.2"
+version = "0.3"
 optional = true
 
 [dependencies.gfx_window_sdl]
@@ -142,4 +142,3 @@ rand = "0.3"
 genmesh = "0.4"
 noise = "0.1"
 image = "0.13"
-winit = "0.6"

--- a/src/window/metal/Cargo.toml
+++ b/src/window/metal/Cargo.toml
@@ -14,7 +14,7 @@
 
 [package]
 name = "gfx_window_metal"
-version = "0.2.0"
+version = "0.3.0"
 description = "Metal window for gfx-rs"
 homepage = "https://github.com/gfx-rs/gfx"
 repository = "https://github.com/gfx-rs/gfx"

